.ci/windows.groovy

```diff
@@ -75,6 +75,9 @@ pipeline {
     }
     stage('Lint'){
       options { skipDefaultCheckout() }
+      when {
+        expression { return false }
```

Comment:
I'm not following this. Should the stage even be in the pipeline if it's never run?

Reply:
The idea of this pipeline was initially to be similar to the `Jenkinsfile` then we had to remove certain stages, I could remove it although I was a bit skeptical about the merge from Jenkinsfile if there are new changes. 


Short story, Jenkinsfile is too big, we decided to create another MBP to iterate with the windows workers supports, and for such we duplicated the Jenkinsfile and removed some stages.

At the moment it's not production ready but providing us feedback to iterate with the pipeline until we decid what to do